hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/AuthenticationBaseMessageTask.java

```diff
@@ -134,6 +134,12 @@ public void processMessage() throws Throwable {
             return;
         }
         String uuid = getUuid();
+        if (clientMessage.getCorrelationId() < clientEngine.getLastAuthenticationCorrelationId(uuid)) {
+            String message = "Server already processed authentication from client with uuid " + uuid
+                    + ". Not applying requested ownership change to " + clientEngine.getThisUuid();
+            logger.info(message);
```

Comment:
Previous one was warning but this is info.  https://github.com/hazelcast/hazelcast/pull/9506/files#diff-6bae815677da05c4ca3cecc2f364fe2bR55